src/ethtool/Makefile

```diff
@@ -0,0 +1,19 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = $(ETHTOOL)
+DERIVED_TARGET = $(ETHTOOL_DBG)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Obtaining the ethtool debian package
+	rm -rf ethtool*
+	git clone https://salsa.debian.org/kernel-team/ethtool/
+	pushd ethtool
+	git checkout tags/debian/1%$(ETHTOOL_VERSION_BASE)-1
+	# Build package
+	dpkg-buildpackage -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS)
```

Comment:
need admin_dir, can you check other Makefile example.

Reply:
Fixed